ops/cloud-deployment/main.bicep

```diff
@@ -1,241 +1,214 @@
+@description('Private Endpoint Subnet Name. USTP Required')
+param privateEndpointSubnetName string = 'snet-${appName}-private-endpoints'
+@description('Private Endpoint Subnet address prefix. USTP Required')
+param privateEndpointSubnetAddressPrefix string = '10.10.12.0/28'
+
+@description('Flag: Deploy Bicep config for webapp. False on slot deployments . Flexion and USTP Required')
 param deployWebapp bool = true
 param webappName string = '${appName}-webapp'
-param webappResourceGroupName string
+@description('Webapp Subnet Name. USTP Required')
 param webappSubnetName string = 'snet-${webappName}'
+@description('Webapp Subnet address prefix. USTP Required')
 param webappSubnetAddressPrefix string = '10.10.10.0/28'
-param webappPrivateEndpointSubnetName string = 'snet-${appName}-pep'
-param webappPrivateEndpointSubnetAddressPrefix string = '10.10.11.0/28'
-param webappPlanName string = 'plan-${webappName}'
-@description('Plan type to determine webapp service plan Sku')
+@description('Plan type to determine webapp service plan Sku. Flexion and USTP Required')
 @allowed([
   'P1v2'
   'B2'
   'S1'
 ])
-param webappPlanType string
+param webappPlanType string = 'P1v2'
 
-@description('Set to true to deploy api module resources. This should be set to false for Azure slot deployments.')
+@description('Flag: Deploy Bicep config for Azure function. False on slot deployments . Flexion and USTP Required')
 param deployFunctions bool = true
-param apiName string = '${appName}-node-api'
-param apiFunctionsResourceGroupName string
-param apiFunctionsSubnetName string = 'snet-${apiName}'
-param apiFunctionsSubnetAddressPrefix string = '10.10.12.0/28'
-param apiPrivateEndpointSubnetName string = 'snet-${appName}-pep'
-param apiPrivateEndpointSubnetAddressPrefix string = '10.10.11.0/28'
-param apiPlanName string = 'plan-${apiName}'
+param functionName string = '${appName}-node-api'
+@description('Function Subet Name. USTP Required')
+param functionSubnetName string = 'snet-${functionName}'
+@description('Function Subnet Address Prefix. USTP Required')
+param functionSubnetAddressPrefix string = '10.10.11.0/28'
 @description('Plan type to determine functionapp service plan Sku')
 @allowed([
   'P1v2'
   'B2'
   'S1'
 ])
-param apiPlanType string
+param functionPlanType string = 'P1v2'
+
 
 param privateDnsZoneName string = 'privatelink.azurewebsites.net'
-param privateDnsZoneResourceGroup string = resourceGroup().name
+param privateDnsZoneResourceGroup string = networkResourceGroupName
+@description('DNS Zone Subscription ID. USTP uses a different subscription for prod deployment. USTP Required')
 param privateDnsZoneSubscriptionId string = subscription().subscriptionId
 
 @description('Name of deployment slot for frontend and backend')
 param slotName string = 'staging'
-param azHostSuffix string = '.net'
+param azHostSuffix string = '.us'
 
-@secure()
-param databaseConnectionString string = ''
+@description('SQL Server Name. Flexion and USTP Required')
```

Comment:
If this is required, why does it have the default value? Could we not just remove the default and not need the description at all?